spec/datetimeformat.html

```diff
@@ -363,12 +350,13 @@
       </p>
 
       <emu-alg>
-        1. Let _dtf_ be this DateTimeFormat object.
-        1. ReturnIfAbrupt(_dtf_).
-        1. If the [[boundFormat]] internal slot of this DateTimeFormat object is *undefined*, then
+        1. Let _dtf_ be *this* value.
+        1. If Type(_dtf_) is not Object, throw a *TypeError* exception.
+        1. If _dtf_ does not have a [[boundFormat]] internal slot, throw a *TypeError* exception.
+        1. If the [[boundFormat]] internal slot of _dtf_ is *undefined*, then
           1. Let _F_ be a new built-in function object as defined in <emu-xref href="#sec-datetime-format-functions"></emu-xref>.
-          1. The value of _F_’s *length* property is 1.
-          1. Let _bf_ be BoundFunctionCreate(F, «*this* value»).
+          1. The value of _F_’s `length` property is 1.
```

Comment:
This change keeps appearing? I looked through the ecma-262 commits and don't see a new syntax for backticks.